smithy-aws-protocol-tests/model/rpcV2/empty-input-output.smithy

```diff
@@ -0,0 +1,189 @@
+$version: "2.0"
+
+namespace aws.protocoltests.rpcv2Cbor
+
+use smithy.protocols#rpcv2Cbor
+use smithy.test#httpRequestTests
+use smithy.test#httpResponseTests
+
+
+@httpRequestTests([
+    {
+        id: "no_input",
+        protocol: rpcv2Cbor,
+        documentation: "Body is empty and no Content-Type header if no input",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Accept": "application/cbor",
+        },
+        forbidHeaders: [
+            "Content-Type",
+            "X-Amz-Target"
+        ]
+        method: "POST",
+        uri: "/service/RpcV2Protocol/operation/NoInputOutput",
+        body: ""
+    },
+    {
+        id: "no_input_server_allows_accept",
+        protocol: rpcv2Cbor,
+        documentation: "Servers should allow the Accept header to be set to the default content-type.",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Accept": "application/cbor",
+            "Content-Type": "application/cbor"
+        }
+        method: "POST",
+        uri: "/service/RpcV2Protocol/operation/NoInputOutput",
+        body: "",
+        appliesTo: "server"
+    },
+    {
+        id: "no_input_server_allows_empty_cbor",
+        protocol: rpcv2Cbor,
+        documentation: "Servers should accept CBOR empty struct if no input.",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Accept": "application/cbor",
+            "Content-Type": "application/cbor"
+        }
+        method: "POST",
+        uri: "/service/RpcV2Protocol/operation/NoInputOutput",
+        body: "v/8=",
+        appliesTo: "server"
+    },
+    {
+        id: "NoInputServerIngoresUnexpectedFields",
+        protocol: rpcv2Cbor,
+        documentation: "Servers should accept CBOR empty struct if no input.",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Accept": "application/cbor",
+            "Content-Type": "application/cbor"
+        }
+        method: "POST",
+        uri: "/service/RpcV2Protocol/operation/NoInputOutput",
+        body: "v/8=",
+        appliesTo: "server"
+    }
+])
+@httpResponseTests([
+    {
+        id: "no_output",
+        protocol: rpcv2Cbor,
+        documentation: "Body is empty and no Content-Type header if no response",
+        body: "",
+        bodyMediaType: "application/cbor",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+        },
+        forbidHeaders: [
+            "Content-Type"
+        ]
+        code: 200,
+    },
+    {
+        id: "no_output_client_allows_accept",
+        protocol: rpcv2Cbor,
+        documentation: "Servers should allow the accept header to be set to the default content-type.",
```

Comment:
Says "Servers" but is a client test. I don't think `Accept` is relevant for responses either.

Reply:
I agree, we shouldn't be testing `Accept` headers in response. Will delete this test.